dependencies.gradle

```diff
@@ -15,7 +15,7 @@ project.ext.versions = [
     cli: '1.3.1',
     csv: '1.2',
     lang: '3.4',
-    reflections: '0.9.10',
+    fastclasspath: '2.8.0',
```

Comment:
Can we upgrade to the latest - 2.9.4? See https://github.com/lukehutch/fast-classpath-scanner/releases for release notes.